### ISSUE TITLE
How can i customize my config file ?

### ISSUE THREAD

**Issue by cnshn89** [tailwindcss/tailwindcss#789]
Hello everyone ,

I'm coding with Django on VSCODE. According to this documentation , i applied 1 ,2 and 3 section (https://tailwindcss.com/docs/installation) and only i executed these codes  on cmd ,

**npm install tailwindcss
npx tailwind init [filename]**

then created node modules folder and others. 

![image](https://user-images.githubusercontent.com/34402743/54638725-c4494300-4a9c-11e9-9dae-f02e9f49fc61.png)

Is that enough for my project ? If it's enough , how can i customize my config file ? I'm so confused.

Best Regards,

Thanks.

**Comment by devcircus** [comment]
You shouldn't be creating the node modules folder. If npm install worked properly, it should be created already. 

After following the "installation" instructions, if you want to customize, follow the "configuration" page. You can skip running "init" since you've already made it.

**Comment by cnshn89** [comment]
Yes , that command created node_moduls folder . I did not add manually. 
- npm install tailwindcss

Should I run that code on cmd ?

./node_modules/.bin/tailwind init MYCSS

**Comment by adamwathan** [comment]
The config file is right there in the screenshot, `tailwind.js`. To customize it, just open the file and change it in your editor.

You also need to create a CSS file like outlined here: https://tailwindcss.com/docs/installation#3-use-tailwind-in-your-css

And then follow the instructions to compile your CSS using your build tool of choice: https://tailwindcss.com/docs/installation#4-process-your-css-with-tailwind

**Comment by cnshn89** [comment]
Ok, I created my css file as mycss.css and used here: https://tailwindcss.com/docs/installation#3-use-tailwind-in-your-css.


**Is that a problem ?**

![image](https://user-images.githubusercontent.com/34402743/54691991-4c7b2700-4b35-11e9-9453-79838cc12bf6.png)



**Compile my css, Is that true ?**

![image](https://user-images.githubusercontent.com/34402743/54692225-b3004500-4b35-11e9-83f1-3a851a4900fd.png)

**Comment by adamwathan** [comment]
That is just your editor warning you because "@tailwind" is not a real CSS feature, don't need to worry about that.

That command is not quite enough to compile your CSS, you need to specify an output path as well as your config path:

```
npx tailwind build mycss.css -c tailwind.js -o output.css
```

You might need to rename `tailwind.js` to `tailwind.config.js` because `npx` sometimes has issues with the default name on Windows.

**Comment by adamwathan** [comment]
Moving this to our `discuss` repo since this isn't an issue with the framework, just a request for help.
